zipline/pipeline/factors/factor.py

```diff
@@ -1065,6 +1066,8 @@ def top(self, N, mask=NotSpecified, groupby=NotSpecified):
         -------
         filter : zipline.pipeline.filters.Filter
         """
+        if N == 1:
+            return self.maximum(mask=mask, groupby=groupby)
```

Comment:
Looks like `maximum` is not defined

Reply:
Oh, oops, I deleted this before pushing since I didn't want to add a public entrypoint for this yet; forgot that was how I was actually invoking the method here.